src/clr-angular/button/button-group/_button-group.clarity.scss

```diff
@@ -15,6 +15,27 @@
     .btn {
       margin: 0;
       vertical-align: top;
+      overflow: visible;
+
+      label {
+        height: 100%;
+      }
+
+      input[type='radio']:focus,
+      input[type='checkbox']:focus {
+        appearance: none;
```

Comment:
The idea is great. You may also need a `-webkit-appearance: none;`. I tried to reproduce it in a stackblitz and got the native input remaining too: 
<img width="162" alt="screen shot 2018-12-11 at 11 29 39 am" src="https://user-images.githubusercontent.com/1798828/49791099-63d99580-fd38-11e8-9a40-4c6dc12a6ba5.png">

Please, check it here - https://stackblitz.com/edit/jeeyuns-awesome-focus-fix?file=src/app/app.component.css

nit: Some screens with the active/primary inputs focused would be helpful, too.

Reply:
We use autoprefixer to not have to worry about all these vendor prefixes. That's why you see the problem in StackBlitz but Jeeyun doesn't have it in the project.